### PR TITLE
Default components

### PR DESCRIPTION
**Pull request by sonofabear**
Why? Because we need some default components for building raygun templates.

Added 5 components:
-Banner
-Text
-CTA
-Contact
-Card List

Also added some mixins, default styling, and more goodies to help get a new project off the ground faster.

### CODE REVIEW

File: app/views/admin/_edit_buttons.html.slim [hunk 1]
```diff
@@ -1,3 +1,13 @@
 ol.edit-buttons
   li
+    = link_to "Banner", atomic_cms.edit_component_path("banner"), class: "button"
+  li
+    = link_to "Card Group", atomic_cms.edit_component_path("card_group"), class: "button"
+  li
+    = link_to "Text Block", atomic_cms.edit_component_path("text_block"), class: "button"
+  li
     = link_to "Text Block", atomic_cms.edit_component_path("text_block"), class: "button"
```
Comment (samcdavid) [inline app/views/admin/_edit_buttons.html.slim]: Does text block need to be in here twice?

Comment (sonofabear) [inline app/views/admin/_edit_buttons.html.slim]: no, removed.

Comment (Mrjaco12) [inline app/views/admin/_edit_buttons.html.slim]: Double your text block. Double your fun.

File: app/components/contact_form_component.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+class ContactFormComponent < AtomicAssets::Component
+  def edit
+    rtn = cms_fields(field_types)
+    rtn << h.component(:contact_form, field_previews).render
+    rtn.html_safe
+  end
+
+  protected
+
+  def field_previews
```
Comment (Mrjaco12) [inline app/components/contact_form_component.rb]: Metrics/MethodLength: Method has too many lines. [7/5]

**Comment by Mrjaco12** [comment]
Oh man. So many double comments....

**Comment by samcdavid** [comment]
:+1:

File: .rubocop.yml [hunk 3]
```diff
@@ -234,7 +237,7 @@ Lint/AssignmentInCondition:
 
 Style/InlineComment:
   Description: Avoid inline comments.
-  Enabled: true
+  Enabled: false
```
Comment (samcdavid) [inline .rubocop.yml]: Enabling this cop caused RuboCop fines when disabling cops in component logic.

**Comment by samcdavid** [comment]
Hmm, It looks like adding in the disables doesn't not remove the previous findings in hound :frowning:

**Comment by mirceacr** [comment]
:+1:

**Comment by squaresurf** [comment]
:+1:

**Comment by handerson** [comment]
:+1:

**Pull request merged**